runestone/codelens/css/pytutor.css

```diff
@@ -221,7 +221,7 @@ div.ExecutionVisualizer #pyStdout {
   font-size: 12pt;
   padding: 4px;
   font-family: Andale mono, monospace;
-
+  --bodyFont: #e93f34;
```

Comment:
This is incorrect.  `--bodyFont` has been defined as a variable in `runestone-custom-sphinx-bootstrap.css`. To **use** a variable in another css rule you need to use the correct syntax.

In this case that should be something like: `color: var(--bodyFont);

I bet if you look at the output from `npm run build` it would tell you that this is a syntax error in your css file.

Reply:
@bnmnetp , I tried following this feedback, but I did not receive any error feedback from "npm run build". I tried doing a simple test by changing line 224 to: " background-color: blueviolet; ", however I am not seeing any changes at all to the Codelens print output. I wasn't very specific in my pull request message but it seems that any changes my team makes in RunestoneComponents does not get updated. Whenever we make a change in components in our local environment, we first save the change in the environment, npm run  build, ./dbuild the book, then refresh the local website page. Is there something we are missing in our process?

Reply:
For example, even if we change the value of --bodyFont in line 27 of runestone-custom-sphinx-bootstrap.css to a value other than white, we still cannot see any changes to CSS in the website. The syntax should already be good, since this is the current state of the file in the master repository.

Reply:
You can debug this somewhat by using the developer tools window.  You can use ctrl-p or cmd-p on the Mac to select a file you want to view such as pytutor.css you can then see if your changes are there.

Also doing an inspect element on the font in codelens and then checking to see that you are changing the correct file that controls the colors in this particular codlens would be useful.

You should make sure that the RunestoneComponents are installed  in developer mode in your container.  Shelling in to the container and then running which Runestone or pip list should show you that you have the Runestone components coming from /srv/RunestoneComponents